### DISCUSSION TITLE
no way jose

### DISCUSSION THREAD

**Discussion by rwg100** [3kh0/ext-remover#851] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
im shidding myself really hard because i power washed the Chromebook and now its in a million pieces, i think i should have stuck to the 1500 psi pressure washer but another idea is just take a 2jz and a turbo and add boost until it blows up please help

**Comment by funkymokey** [comment]
just pretend the toyota jz engine is the chromebook

**Comment by rwg100** [comment, marked as answer]
guys nevermind i ls swapped the pressure washer engine with an ls3 and may or may not have made the water blow the fuck up

**Comment by Su4o** [comment]
Oh no! Here's a solution. 

Requirements
1. Money (if you don't have it, no profit for u) 
2. Ebay store near u
3. Your psi pressure thing
-----------------------------------------
First, buy a phone to search for something. (buy a new phone instead of using your old phone, cause you need a new phone, like look at it rq..) 

Next, Use your wifi to go onto [https://ebay.com](https://ebay.com). (if you don't have wifi, i suggest you to instead buy a cell phone tower thingamajig and a sim card.)

Then, Search for the following "Chromebook"

Afterwards, buy the chromebook _using your ipad._  (if you don't have credit, use debit. No debit? no profit. :wah: ) 

Now, once you get your chromebook, **correctly** use your psi pressure to 'powerwash' your chromebook

DONE! Profit.

**Reply by rwg100** [reply]
instructions unclear i created a time machine and rejected some retard with a mustache from art school in 1907 and started his political career in germany

**Reply by Su4o** [reply]
uh oh. sorry but go to microsoft tech support, this is not my problem anymore
